src/components/Slider/index.tsx

```diff
@@ -29,6 +32,8 @@ interface SliderProps {
   customStyles: ChakraProps
   swiperStyles: CSSObject
   isLoading: boolean
+  withHero: boolean
+  simpleCard: boolean
```

Comment:
I had seen a couple of design principles in [Chakra](https://chakra-ui.com/docs/principles). Don't you think it's better to have similar principle in terms of consistency?

> **Naming Props:** We all know naming is the hardest thing in this industry. Generally, ensure a prop name is indicative of what it does. Boolean props should be named using auxiliary verbs such as does, **has**, **is** and **should**. For example, Button uses **_isDisabled_**, **_isLoading_**, etc.